department-of-veterans-affairs/caseflow
# Justin/appeals 42114

Resolves [Correspondence Controller needs to be Split Apart: Queue](https://jira.devops.va.gov/browse/APPEALS-42114)

# Description
Moved queue methods from the CorrespondenceController to a new child CorrespondenceQueueController. Created a spec file to be worked in a later ticket. Added a test to verify the new routing in this spec file.

## Acceptance Criteria
- [ ] Move Correspondence Queue code into its own child controller

## Testing Plan

1. Log in as Jolly Postman and go to your queue
2. Open the RailsPanel extension on Chrome
3. With the rails tool open, switch views to the correspondence cases
4. The rails panel should show the queue method being hit on the new queue controller. The following method should be seen in this step:
- CorrespondenceQueueController#correspondence_cases

Note: Previously, this would hit the following route: CorrespondenceController#correspondence_cases. This ticket was to create the new files so that the CorrespondenceQueueController#correspondence_cases route would be hit instead.
